src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/Provider.cs

```diff
@@ -29,7 +29,7 @@ internal Provider(long ukPrn, string name, DateTime created, DateTime updated)
             DraftApprenticeshipDetails draftApprenticeshipDetails,
             UserInfo userInfo)
         {
-            return new Cohort(provider, accountLegalEntity, transferSender, draftApprenticeshipDetails, Party.Provider, userInfo);
+            return new Cohort(provider, accountLegalEntity, Party.Provider, userInfo);
```

Comment:
because we are positively NOT expecting to receive transferSenderId or a DraftApprenticeshipDetails. Do we either explicitly throw an exception at this point or extend the ICohortOriginator interface so it supports a method with the signature

Cohort CreateCohort(Provider provider,  AccountLegalEntity accountLegalEntity, UserInfo userInfo);

Doing this would mean we would have to implement this on the Employer implementation, which would throw an exception if called

Reply:
As suggested, created a new method in IOriginator interface.